# base app exception
class Error(Exception):
    pass


class ResourceNotFoundError(Exception):
    pass
